ui/app/components/ui/searchable-item-list/item-list/item-list.component.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+export default function SearchableItemList ({
+  results = [],
```

Comment:
I personally prefer use of default parameters: fewer lines of code, and having the variable declaration and default setting in the same place results in less moving around when reading the code (with these functional components the `defaultProps` object can be many lines away from the function signature)

Reply:
Does the fact that "Most other components use defaultProps to set default props instead of this syntax" mean that there is already an established preference on the team? If so, I'll follow that, but if not I'll campaign for using default parameters.

Reply:
I prefer default parameters for the same reasons. I thought there were reasons to prefer `defaultProps` regardless, but I can't recall any, and the rest of the team seems OK with using default parameters as well. Plus the React team is planning on deprecating `defaultProps` for functional components, so I guess we should consider the use of default parameters to be convention from this point forward.